docs/reference/api.yaml

```diff
@@ -1294,6 +1294,82 @@ paths:
           description: App not found
           schema:
             $ref: "#/definitions/ErrorMessage"
+  /1.0/apps/{app}/start:
+    parameters:
+      - name: app
+        in: path
+        required: true
+        type: string
+        minLength: 1
+        description: App name.
+    post:
+      operationId: AppStart
+      description: Start App.
+      tags:
+        - app
+      security:
+        - Bearer: []
+      consumes:
+        - application/json
+      parameters:
+        - name: process
+          in: formData
```

Comment:
@CarlosRoGuerra I believe that `in: formData` wont work, cause there are a json body